functions/Install-DbaSqlWatch.ps1

```diff
@@ -0,0 +1,231 @@
+
+    .PARAMETER Confirm
+        Prompts to confirm actions
+
+    .PARAMETER WhatIf
+        Shows what would happen if the command were to run. No actions are actually performed.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: SqlWatch
+        Author: Ken K (github.com/koglerk)
+        Website: https://sqlwatch.io
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Install-DbaSqlWatch
+
+    .EXAMPLE
+        Install-DbaSqlWatch -SqlInstance server1 -Database master
+
+        Logs into server1 with Windows authentication and then installs SqlWatch in the master database.
+
+    .EXAMPLE
+        Install-DbaSqlWatch -SqlInstance server1\instance1 -Database DBA
+
+        Logs into server1\instance1 with Windows authentication and then installs SqlWatch in the DBA database.
+
+    .EXAMPLE
+        Install-DbaSqlWatch -SqlInstance server1\instance1 -Database master -SqlCredential $cred
+
+        Logs into server1\instance1 with SQL authentication and then installs SqlWatch in the master database.
+
+    .EXAMPLE
+        Install-DbaSqlWatch -SqlInstance sql2016\standardrtm, sql2016\sqlexpress, sql2014
+
+        Logs into sql2016\standardrtm, sql2016\sqlexpress and sql2014 with Windows authentication and then installs SqlWatch in the master database.
+
+    .EXAMPLE
+        $servers = "sql2016\standardrtm", "sql2016\sqlexpress", "sql2014"
+        $servers | Install-DbaSqlWatch
+
+        Logs into sql2016\standardrtm, sql2016\sqlexpress and sql2014 with Windows authentication and then installs SqlWatch in the master database.
+
+    .EXAMPLE
+        Install-DbaSqlWatch -SqlInstance sql2016 -Branch development
+
+        Installs the dev branch version of SqlWatch in the master database on sql2016 instance.
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = "Low")]
+    param (
+        [Parameter(Mandatory, ValueFromPipeline)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [string]$Database = "master",
+        [string]$LocalFile,
+        [switch]$Force,
+        [switch]$EnableException
+    )
+
+    begin {
+
+        $DbatoolsData = Get-DbatoolsConfigValue -FullName "Path.DbatoolsData"
+        $tempFolder = ([System.IO.Path]::GetTempPath()).TrimEnd("\")
+        $zipfile = "$tempFolder\SqlWatch.zip"
+
+        if ($LocalFile -eq $null -or $LocalFile.Length -eq 0) {
```

Comment:
This can actually be taken care of by `-not $LocalFile` or using our internal command `Test-Bound`.  `-not` is awesome and takes care of empty, null, false 💯

Reply:
Done.